### ISSUE TITLE
[FEATURE] Requesting BDFR work with Catbox.Moe albums

### ISSUE THREAD

**Issue by CZAlmon** [Serene-Arc/bulk-downloader-for-reddit#858]
- [ ] I am requesting a feature. 

## Description

Due to some websites changing rules on images, some other services are starting to be used more. I would ask if BDFR could be expanded to be able to download albums from catbox.moe - https://catbox.moe/tools.php

I think single images that are directly linked are/can be downloaded. But I know an album cannot be downloaded. This is the error I get: 

[2023-05-12 15:44:06,020 - bdfr.downloader - DEBUG] - Attempting to download submission 
[2023-05-12 15:44:07,765 - bdfr.downloader - ERROR] - Could not download submission : No downloader module exists for url https://catbox.moe/c/

**Comment by OMEGARAZER** [comment]
Can you please try with the Dev branch as this was added [here](https://github.com/aliparlakci/bulk-downloader-for-reddit/commit/72d5b1acaf9e756117363745c0bfd148d51a7a46)
You can use `pip install git+https://github.com/aliparlakci/bulk-downloader-for-reddit@development` to install the dev branch.

**Comment by CZAlmon** [comment]
Sorry, I am not 100% familiar with GIT. I ran your command and have it downloaded/installed. I ran the same command I used before and it gave me the same error. I have a feeling it used the old BDFR and not the dev branch version.

How exactly do I run the dev branch version of BDFR?

**Comment by CZAlmon** [comment]
Also, is it too late or too much to also ask for https://www.imgchest.com/ ? Should I open a new feature request for it? I got the same error for imgchest.com as I did for catbox.moe.

https://www.imgchest.com/p/ro24aogylj5

**Comment by OMEGARAZER** [comment]
> Sorry, I am not 100% familiar with GIT. I ran your command and have it downloaded/installed. I ran the same command I used before and it gave me the same error. I have a feeling it used the old BDFR and not the dev branch version.
> 
> How exactly do I run the dev branch version of BDFR?

that command should install the dev version over any older version installed. Did you use pip initially to install it or a different tool like pipx?



> Also, is it too late or too much to also ask for https://www.imgchest.com/ ? Should I open a new feature request for it? I got the same error for imgchest.com as I did for catbox.moe.
> 
> https://www.imgchest.com/p/ro24aogylj5

Yes, please open a new feature request for this, I'll take a look into it today.

**Comment by CZAlmon** [comment]
I don't think I used pipx. I have always just used pip, but it was several months ago. I didn't install it initially using git, I know that for sure. I used just python and pip. I had to download git to use your command to download the dev branch.

Here is my window. I ran your code again. I checked bdfr's version. and I ran the command I use again to test a link and still got the same error that there is no downloader module available.

![image](https://github.com/aliparlakci/bulk-downloader-for-reddit/assets/13081748/e6ed789b-a8ea-41f2-854d-1c57b4e726fc)
